cs/vaccination-co/index.js

```diff
@@ -685,10 +685,18 @@ function prepare_charts() {
                 dataLabels: {
                     enabled: true,
                     format: '{point.name}'
-                }
+                },
+				point: {
```

Comment:
Please here watch out for indentation, we prefer to use spaces, maybe the tabs are making this look a bit weird